index.js

```diff
@@ -3,5 +3,15 @@ const banks = require('./banks');
 module.exports = {
   getBanks: function() {
     return banks;
+  },
+  getBank: code => {
+  	let OneBank = null
+  	let allBanks = [...banks]
+  	try{
+  		allBanks.forEach( singleBank => { singleBank.code === code ? OneBank = singleBank : "Invalid bank code" })
```

Comment:
Instead of using the `.forEach` method, you should make use of the [.filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method since all we want to do is retrieve just one bank from the list of banks.